data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg

```diff
@@ -590,6 +590,33 @@ Besides... I want my brother back."
             result=defeat
         [/endlevel]
     [/event]
+
+    [event]
+        name=scenario_end
+        [filter_condition]
+            [proceed_to_next_scenario]
+            [/proceed_to_next_scenario]
+        [/filter_condition]
```

Comment:
I consider this filter condition to be ugly and prefer to use `name=victory` when you don't actually need the event to be synced (ie, when you're working on a single-player campaign). I already reverted the non-LoW cases in mainline (except those that didn't rely on the condition).